client/style.css

```diff
@@ -1,5 +1,18 @@
-.nav__item--main li a {
-    cursor: pointer;
+
+/*===========================
+     --- Base Styles ---
+===========================*/
+html {
```

Comment:
You forgot to put margin and padding to 0 and box-sizing to border-box. This is a good practice. Please revise this:
```css
* {
margin: 0;
padding: 0;
border-box: inherit;
}

body {
box-sizing: border-box;
}
```